worlds/yachtdice/docs/setup_en.md

```diff
@@ -6,10 +6,10 @@
 
 ## Playing the game
 Open the Yacht Dice website. There are two options:
-- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser. 
-- Cruise over to the [Yacht Dice website](https://yacht-dice-ap.netlify.app/). This also works on mobile. If the website is not available, use the first option.
+- Cruise over to the [Yacht Dice Website](https://yacht-dice-ap.netlify.app/). This is the easiest option. If the website is unavailable, use the next option.
+- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser.
 
 Press Archipelago, and after logging in, you are good to go. The website has a built-in client, where you can chat and send commands.    
-Both options also have an "offline" play option to try out the game without having to generate a game first.
+The website also have an "offline" play option to try out the game without having to generate a game first.
```

Comment:
Why this change? Is it because the download is "technically" already offline? I'd still say it's fair to say you can do it offline with the downloaded version tbh.


If you want to keep this change though you need to change have to has.

Reply:
Fair! I've changed it to:
Both options also have a "Solo play" mode
Which I think makes sense because that's what the mode is called on the website.